eureka/lib/readECF.py

```diff
@@ -95,8 +95,15 @@ def __init__(self, params):
 
   # :::: And now, Chris' code ::::
   # load all parameters
+    #parlist=[]
     for i, parname in enumerate(params):
       exec("self.{pname} = Param(parname[1:])".format(pname  = parname[0]))
+      #parlist.append(parname[0])
```

Comment:
If you no longer need these commented out edits you made, best to delete them

Reply:
In all the comment cases, I was afraid of deleting something until someone else verified my changes worked. (: Will delete now!

Reply:
The nice thing is the edits are at least now saved in GitHub, so you can delete them and if ever you need them again you can try to search back through your commit history.